lib/timer.js

```diff
@@ -0,0 +1,11 @@
+var gauge = require('./gauge');
+var types = require('./types');
+
+module.exports = function(metricsClients) {
+
+  var g = gauge(metricsClients, types.T);
+
+  return function(key, tags, value, cb) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)